### DISCUSSION TITLE
Compiled model (source)

### DISCUSSION THREAD

**Discussion by Mutaz94** [metrumresearchgroup/mrgsolve#871] [General]
Hi All, 

By any chance, can one have access to the compiled C++ code for mrgsolve model? I know that mrgsolve model would be translated to something before `mread()`  can read it. Does that make sense? 

Mutaz

**Comment by kylebaron** [comment]
You can pass a (local) directory name to `mread()` as the `soloc` argument and it will have all the actual cpp and header files there. It does link back to some headers inside the package too.

**Reply by Mutaz94** [reply]
Awesome!! Yes, it dumps all the files that I want to take look at. Thanks!!
